### ISSUE TITLE
Extend the 'pokemons' table with additional values

### ISSUE THREAD

**Issue by grudzik** [ArkadiuszKubiak/kursgrupaa#11]
The 'Pokemons' table's missing some values, when compared:
https://raw.githubusercontent.com/Biuni/PokemonGO-Pokedex/master/pokedex.json

and 
class PokemonRecord constructor(
    @PrimaryKey @ColumnInfo(name = "num") var num: String,
    @ColumnInfo(name = "name") var name: String,
    @ColumnInfo(name = "img") var img: String,
    @ColumnInfo(name = "height") var height: String,
    @ColumnInfo(name = "weight") var weight: String,
    @ColumnInfo(name = "candy") var candy: String,
    @ColumnInfo(name = "candy_count") var candy_count: Int,
    @ColumnInfo(name = "egg") var egg: String,
    @ColumnInfo(name = "spawn_chance") var spawn_chance: Double,
    @ColumnInfo(name = "spawn_time") var spawn_time: String
)

Extend the DB with these values, it can be done in other tables, such as:

PokeWeaknesses => id, pokeId, weaknessID

Weaknesses = id, weaknessName

bla bla; generally rely on generated stuff, which can be found in:
com/example/homework1/course/models/Pokemon.kt

**Comment by grudzik** [comment]
Should be done after #4 is closed though.

**Comment by grudzik** [comment]
I'll do it with #4, since I have spare time

**Comment by grudzik** [comment]
Need to decide though, what's really needed.

**Comment by grudzik** [comment]
kinda not needed now
